### PR TITLE
Add Git Bash on Windows disclaimer

### PR DESCRIPTION
**Pull request by arleytm**

### CODE REVIEW

File: get-started/troubleshooting.md [hunk 1]
```diff
@@ -533,6 +533,8 @@ This setting requires at least `@sap/cds-mtx@2.6.2`.
 
 - Make sure to use the latest version of the [Cloud MTA Build Tool (MBT)](https://sap.github.io/cloud-mta-build-tool/).
 - Consult the [Cloud MTA Build Tool documentation](https://sap.github.io/cloud-mta-build-tool/usage/) for further information, for example, on the available tool options.
+- Please note that Git Bash on Windows, despite offering a Unix-like environment, may encounter compatibility issues with certain scripts or tools due to its hybrid nature between Windows and Unix systems.
+When using Azure DevOps pipelines, we recommend testing and verifying all functionalities in the native Windows Command Prompt (cmd.exe) or PowerShell for optimal compatibility.
```
Comment (renejeglinsky) [inline get-started/troubleshooting.md]: Is that really an issue with the MTA build? Also I'm not sure if we're allowed to mention the product of a competitor. Azure DevOps pipelines doesn't sound like BTP deployment, or?

Comment (arleytm) [reply get-started/troubleshooting.md]: The related issue was internally reported for a failed MTA build on Azure. Although it only affects the build of a specific customer, similar issues could arise in other non-MTA-related components. Therefore, before you comment, I have created a dedicated section.

File: get-started/troubleshooting.md [hunk 2]
```diff
@@ -706,3 +706,10 @@ For SAP HANA deployment errors see [The HANA section](#how-do-i-resolve-deployme
 :::
 
 <div id="end" />
+
+## CAP on Azure/Windows
+
+Please note that Git Bash on Windows, despite offering a Unix-like environment, may encounter compatibility issues with certain scripts or tools due to its hybrid nature between Windows and Unix systems.
```
Comment (vkozyura) [inline get-started/troubleshooting.md:712]: (1) When using Azure DevOps pipelines  <-- I would remove it - you can build extension-tar also locally on Windows (2) Add something like "for example, problems can occure when building mtxs extension on Windows (locally or in cloud)"

File: get-started/troubleshooting.md [hunk 3]
```diff
@@ -706,3 +706,10 @@ For SAP HANA deployment errors see [The HANA section](#how-do-i-resolve-deployme
 :::
 
 <div id="end" />
```
Comment (renejeglinsky) [inline get-started/troubleshooting.md:708]:
```suggestion

```

**Review by renejeglinsky — APPROVED** [review]

**Pull request merged**